setup.py

```diff
@@ -51,7 +51,12 @@
               'snapcraft.plugins',
               'snapcraft.storeapi'],
     package_data={'snapcraft.internal.repo': ['manifest.txt']},
-    scripts=['bin/snapcraft', 'bin/snapcraft-parser'],
+    entry_points={
+        'console_scripts': [
+            'snapcraft = snapcraft.cli.__main__:run',
+            'snapcraft-parser = snapcraft.internal.parser:main',
+        ],
+    },
```

Comment:
Can we delete the files in bin now?
Also, they were doing a very ugly thing calling snapcraft.internal.dirs.setup_dirs. Do we no longer need that? I would be very happy if so.